src/server/tests/helpers.test.ts

```diff
@@ -21,7 +21,7 @@ const mockCheckoutResponse = {
 }
 
 describe("cancelHold", () => {
-  it("should return a success message if hold is cancelled", async () => {
+  it("should return a success message if hold is canceled", async () => {
```

Comment:
Thought this was a typo, but google is telling me both spellings are correct haha

Reply:
Ik lol I wrote all of this code just randomly switching between the british/american spelling which was not ideal– they should all be 'Canceled' now